app/fetchers/route_fetcher.rb

```diff
@@ -3,8 +3,17 @@
 module VCAP::CloudController
   class RouteFetcher < BaseListFetcher
     class << self
-      def fetch(message, readable_route_dataset, eager_loaded_associations: [])
-        dataset = readable_route_dataset.eager(eager_loaded_associations).qualify
+      def fetch(message, readable_space_guids_dataset: nil, eager_loaded_associations: [], omniscient: false)
+        dataset = Route.dataset.eager(eager_loaded_associations).
+                  join(:spaces, id: Sequel[:routes][:space_id]).
```

Comment:
Personally I'm in favor of a different syntax: `id: :routes__space_id` instead of `id: Sequel[:routes][:space_id]`. I find this easier to read.

Reply:
After discussing it with @sethboyles and @moleske, we felt that Sequel[:routes][:space_id] was easier to read for us so i'm leaving this for now. If we want to have a dedicated conversation about it i'm open to it.

Reply:
In the end consistency is more important to me than a specific syntax. This line now uses two different ways to express a "thing", i.e. table `:spaces` and column `Sequel[:routes][:space_id]`.

When I look through the code, I can see several different identifiers (I guess this list is not even complete):
```
:spaces
Sequel[:spaces]
Space.table_name
:apps__space_guid
Sequel[:apps][:space_guid]
"#{AppModel.table_name}__space_guid".to_sym
```

I would prefer if we could agree on a single syntax that's used throughout the codebase. And to me the symbol syntax (i.e. `:spaces` and `:apps__space_guid` is the shortest, simplest and most concise syntax.